modules/bitgo/example/ts/avaxc/send-tokens-eip1559.ts 

```diff
@@ -0,0 +1,64 @@
+import { BitGo } from 'bitgo';
```

Comment:
Same comments as above. Also is there something wrong with this file or is Github acting up with syntax highlighting?